src/parse-options.js

```diff
@@ -13,6 +17,13 @@ const DEFAULT_OPTIONS = {
 function parseOptions (providedOptions) {
   let options = Object.assign({}, DEFAULT_OPTIONS, providedOptions)
 
+  // normalizing options.color to be a boolean or a color value
+  options.color = (options.color === 'false' || options.color === false) ? false : options.color
+
+  if (!(options.color === false || options.color in styles)) {
+    throw new Error('Invalid color option. Set color to true or to one of the value from https://github.com/chalk/ansi-styles#colors')
```

Comment:
Yeah, lets do it later.

Iterating through chalk.styles.styles gives me this:
`reset, bold, dim, italic, underline, inverse, hidden, strikethrough, black, red, green, yellow, blue, magenta, cyan, white, gray, grey, bgBlack, bgRed, bgGreen, bgYellow, bgBlue, bgMagenta, bgCyan, bgWhite`

We might have to whitelist what we show or using url is just fine.